itests/hcatalog-unit/pom.xml

```diff
@@ -89,7 +89,7 @@
     <dependency>
       <groupId>org.apache.hive</groupId>
       <artifactId>hive-standalone-metastore-server</artifactId>
-      <scope>test</scope>
+      <classifier>tests</classifier>
```

Comment:
Could you please explain me why that change is necessary?

Reply:
I wanted to import -> org.apache.hadoop.hive.metastore.utils.TestTxnDbUtil;
It does not allow import  without this change

Reply:
I still wonder: is that works with classifier added but without removing the scope?

Reply:
scope has not been removed. I had removed it by mistake and added it back

Reply:
<img width="984" alt="image" src="https://github.com/apache/hive/assets/1486749/4559ba1f-c0f4-4ce5-b9dd-255e8b3ff694">
<img width="1933" alt="image" src="https://github.com/apache/hive/assets/1486749/4c982108-77a2-4791-a829-94e65d2748a2">


Nice. I love github... :D